newsfragments/3094.misc.rst

```diff
@@ -0,0 +1 @@
+Switch to using PEP570 for positional-only arguments for `~trio.socket.SocketType`'s methods.
```

Comment:
FYI, Sphinx has a `:pep:` role that one can use for PEP links in RST.

Reply:
opened #3160